src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -681,19 +685,86 @@ INFO     Use the following statement to include this module:
  include { SALMON_INDEX } from '../modules/nf-core/salmon/index/main'
 ```
 
+The module is now installed into the folder `modules/nf-core`. Now open the file `workflow/demotest.nf`. You will find already several `include` statements there from the installed modules (`MultiQC` and `FastQC`):
+
+```bash
+
+include { FASTQC  } from '../modules/nf-core/fastqc/main'
+include { MULTIQC } from '../modules/nf-core/multiqc/main'
+```
+
+Now add the above line underneath it:
+
+```bash
+
+include { FASTQC  } from '../modules/nf-core/fastqc/main'
+include { MULTIQC } from '../modules/nf-core/multiqc/main'
+include { SALMON_INDEX } from '../modules/nf-core/salmon/index/main'
+
+```
+
+This makes the module now available in the workflow script and it can be called with the right input data.
+
+<!-- TODO/TO DISCUSS here the user now needs to know about how to get their fasta. We could do this here or add a new point for this above -->
```

Comment:
My logic would be having separate points:
- install the module
- add the inclusion statement and the process
- identify which inputs are required, add (if necessary) a new param, add the input channels with optional channel manipulation
- check the output channels to be used in another process